packages/app/pages/checkout/[network]/[address].tsx

```diff
@@ -198,30 +231,55 @@ const CheckoutPage = ({
         setError(e.message);
         setLoading(false);
       });
-  }, [web3, network, address, price, setSupply, supply, url]);
+  }, [web3, network, address, price, setSupply, supply, customization]);
   return (
     <AppContainer>
-      <AppBackground />
+      <AppBackground 
+        style={{
+          background: (customization.brand_color || "$sand")
+        }}
+      />
       <App>
         <AppOverview>
           <AppHeader>
+            {
+              (customization.logo_cid) ?
+                (<IpfsImage
+                  cid={customization.logo_cid}
+                  height={50}
+                  width={50}
+                />) : null
+            }
             <AppNetworkContainer>{network}</AppNetworkContainer>
           </AppHeader>
           <AppSummaryContainer>
             <ProductSummaryName>
               {name} ({symbol})
             </ProductSummaryName>
             <ProductSummaryAmount>Ξ{price}</ProductSummaryAmount>
+            {
+              (metadata && metadata.thumbnail) ?
+                (<IpfsImage
+                  cid={metadata.thumbnail}
+                />) : null
+            }
           </AppSummaryContainer>
         </AppOverview>
         <AppPayment>
           <PaymentRequestHeader>Pay With Wallet</PaymentRequestHeader>
-          <Button onClick={onBuy} disabled={loading}>
-            Buy ({supply} left)
+          <Button 
+            onClick={onBuy} 
+            disabled={loading}
+            style={{
+              backgroundColor: (customization.accent_color || "$forest")
+            }}
+          >
+            {customization.button_txt || "Buy"} ({supply} left)
```

Comment:
Idea: what if the `button_txt` customization had support for the `{supply}` placeholder which gets replaced with the numerical value of the supply? Then, this could be: 
```javascript
{customization.button_txt?.replace?.(/{supply}/g, supply) || `Buy (${supply} left)`}
```

Reply:
Cool idea! made the change.